### PR TITLE
Optimize fallback function args from static array to dynamic array

### PR DESCRIPTION
**Pull request by yviansu**

### CODE REVIEW

File: runtime-library/libdyntype/dynamic/object.c [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
 #include "libdyntype_export.h"
 #include "type.h"
+#include <math.h>
```
Comment (xujuntwt95329) [inline runtime-library/libdyntype/dynamic/object.c:8]:
```suggestion
```

File: runtime-library/libdyntype/dynamic/object.c [hunk 2]
```diff
@@ -1044,3 +1039,28 @@ dynamic_throw_exception(dyn_ctx_t ctx, dyn_value_t obj)
...
+
+int
+dynamic_get_array_length(dyn_ctx_t ctx, dyn_value_t obj)
+{
+    dyn_value_t length_value = NULL;
+    double length = 0;
+    int ret = -1;
+
+    length_value = dynamic_get_property(ctx, obj, "length");
+    ret = dynamic_to_number(ctx, length_value, &length);
```
Comment (xujuntwt95329) [inline runtime-library/libdyntype/dynamic/object.c:1053]: Why not just invoke QuickJS API?

File: runtime-library/libdyntype/libdyntype_export.h [hunk 3]
```diff
@@ -177,6 +177,24 @@ dyntype_release(dyn_ctx_t ctx, dyn_value_t obj);
...
 
+/*****************************************************************
+*                                                                *
+*                          Section 2                             *
+*                                                                *
+*          Interface exposed to the runtime library              *
+*                                                                *
+*****************************************************************/
+
+/************************** Field access  ***********************/
```
Comment (xujuntwt95329) [inline runtime-library/libdyntype/libdyntype_export.h:188]:
```suggestion
```

File: runtime-library/libdyntype/dynamic/object.c (hunk 2 above)
Comment (yviansu) [reply runtime-library/libdyntype/dynamic/object.c:1053]: Great idea!

**Review by xujuntwt95329 — APPROVED** [review]
LGTM

**Pull request merged**